flixel/util/FlxCollision.hx

```diff
@@ -18,6 +18,7 @@ import flixel.tile.FlxTileblock;
  * @link http://www.photonstorm.com
  * @author Richard Davey / Photon Storm
 */
+@:access(flixel.FlxSprite)
```

Comment:
I haven't done a performance test.  Oh, I see...that's interesting, I had misread drawFrame()'s if statement.  I guess in this case it's properly marked dirty when it needs to be, otherwise this wouldn't be fixing anything.